### ISSUE TITLE
miniconda reinstallation issues in drive C .....

### ISSUE THREAD

**Issue by Jyotisaini-jnu** [ubermag/workshop#11]
Hello Sir..... I am installing ubermag using miniconda3 in windows 10. During reinstallation i faced issues while creating space (as already space is created in drive C) and failing of testing of some files like: mp1 and oommfrun . and many times jupytor is not recognized as external or internal command because of this i reinstalled this many times now i am facing testing failing problem and already created space creating problem. I have attached the word file contains screenshots. 

waiting for your kind reply.

[miniconda issues.docx](https://github.com/ubermag/workshop/files/4807540/miniconda.issues.docx)

Thanks

**Comment by marijanbeg** [comment]
Hi @Jyotisaini-jnu, thank you for your question. It looks like the installation went ok, but there are a few errors reported on the way:

1. As conda suggested, using directory names which contain spaces can be an issue (there is a space in the directory which is named after you). Therefore, it would be better if you can rename the directory you are installing in or just create a new one some else on your C: drive.

2. I can see one test failing - `mpl`, which is a test for plotting 2D planes of fields. When tests complete, you should get a report in the terminal telling you what went wrong. Could you please send us that?

So, in summary, please remove any spaces you have in directory names on you path (e.g. `uber_mag` instead of `uber mag`), make sure the directory you are running tests in is clean from any directories created by previous tests, and run the tests again.

Please let us know how did it go.

**Comment by Jyotisaini-jnu** [comment]
Hello Sir .....All above issues are resolved by making new folder in
another location in drive C.....

Thankyou soo much for your kind response .....

Regards
Jyoti Saini

**Comment by marijanbeg** [comment]
That's great, thank you for letting us know.

Please feel free to raise another issue if you encounter any other problems.
